fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadMgr.java

```diff
@@ -180,59 +183,36 @@ public void beginLoadTaskFromBackend(String dbName, String tableName, String lab
         }
     }
 
-    // for sync stream load
-    public StreamLoadTask createLoadTask(Database db, String tableName, String label, String user, String clientIp,
-                                         long timeoutMillis, boolean isRoutineLoad, long warehouseId)
-            throws StarRocksException {
-        Table table;
-        Locker locker = new Locker();
-        locker.lockDatabase(db.getId(), LockType.READ);
-        try {
-            unprotectedCheckMeta(db, tableName);
-            table = GlobalStateMgr.getCurrentState().getLocalMetastore().getTable(db.getFullName(), tableName);
-        } finally {
-            locker.unLockDatabase(db.getId(), LockType.READ);
-        }
-
+    public StreamLoadTask createLoadTaskWithoutLock(Database db, Table table, String label, String user, String clientIp,
+                                                    long timeoutMillis, boolean isRoutineLoad, long warehouseId) {
         // init stream load task
         long id = GlobalStateMgr.getCurrentState().getNextId();
         StreamLoadTask streamLoadTask = new StreamLoadTask(id, db, (OlapTable) table,
                 label, user, clientIp, timeoutMillis, System.currentTimeMillis(), isRoutineLoad, warehouseId);
         return streamLoadTask;
     }
 
-    public StreamLoadTask createLoadTaskWithoutLock(Database db, String tableName, String label, String user, String clientIp,
-                                                    long timeoutMillis, boolean isRoutineLoad, long warehouseId)
-            throws StarRocksException {
+    private StreamLoadTask createLoadTask(Database db, Table table, String label, String user, String clientIp,
```

Comment:
this interface can be renamed to `createLoadTaskWithoutLock` too because it doesn't retrieve the db/table lock inside the method body.